helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml

```diff
@@ -76,7 +76,7 @@ apiVersion: v1
 kind: Service
 metadata:
   namespace: {{ .Release.Namespace }}
-  name: {{ template "chaos-mesh.name" . }}-chaos-dashboard
+  name: chaos-dashboard
```

Comment:
How about defining the SVC name as a variable?

We also need to update the ingress.

Reply:
How about defining this in `_helpers.tpl`?